<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt4ProjectManager::Internal::MobileAppWizardGenericOptionsPage</class>
 <widget class="QWizardPage" name="Qt4ProjectManager::Internal::MobileAppWizardGenericOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="orientationBehaviorLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Orientation behavior:</string>
       </property>
       <property name="buddy">
        <cstring>orientationBehaviorComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="orientationBehaviorComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>66</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
